libs/common/src/main/java/org/opensearch/common/collect/Map.java

```diff
@@ -32,6 +32,13 @@
 
 package org.opensearch.common.collect;
 
+/**
+ * Java 9 Map
+ *
+ * todo: deprecate and remove w/ min jdk upgrade to 11?
```

Comment:
Could we do that since 3.0.0 has JDK-11 baseline?

Reply:
Yes. I was planning it in a follow up PR.